pom.xml

```diff
@@ -45,7 +45,7 @@
         <!-- Pick up common dependencies for the selected LTS line: https://github.com/jenkinsci/bom#usage -->
         <groupId>io.jenkins.tools.bom</groupId>
         <artifactId>bom-2.361.x</artifactId>
-        <version>1945.v13a_1306588ff</version>
+        <version>1948.veb_1fd345d3a_e</version>
```

Comment:
We need https://github.com/jenkinsci/bom/pull/1897 for the plugin to start with `hpi:run`.